components/server/src/jobs/workspace-gc.ts

```diff
@@ -187,6 +196,9 @@ export class WorkspaceGarbageCollector implements Job {
                 } catch (err) {
                     log.error({ workspaceId: ws.id }, "workspace-gc: failed to purge workspace", err);
                 }
+                log.info({ workspaceId: ws.id }, `workspace-gc: hard deleted a workspace`, {
```

Comment:
Is this the instance id or workspace id? I ask because the workspace id will be redacted.

Reply:
Assumption `ws.id` is the friendly name we show to end-users on the dashboard

Reply:
@kylos101 this will be the workspace ID, and I think it's fine if it's left redacted (we can easily md5 the searched-for workspace ID ourselves).